typography.scss

```diff
@@ -1,193 +1,141 @@
-@import 'pxToEm';
-
-$vui-fontColor: #353535 !default;
-$vui-fontFamily: Arial !default;
-$vui-fontSize: 13px !default;
-$vui-lineHeight: 1.54em !default;
-
-$vui-heading-fontColor: #666666 !default;
-$vui-heading-fontWeight: normal !default;
-$vui-heading-topMargin: 0 !default;
-$vui-heading-rightMargin: 0 !default;
-$vui-heading-bottomMargin: 15px !default;
-$vui-heading-leftMargin: 0 !default;
-$vui-heading-lineHeight: 25px !default;
-$vui-heading1-fontSize: 23px !default;
-$vui-heading2-fontSize: 19px !default;
-$vui-heading3-fontSize: 17px !default;
-$vui-heading4-fontSize: 15px !default;
-
-$vui-emphasis-fontColor: #666666 !default;
-$vui-emphasis-fontStyle: italic !default;
-$vui-emphasis-topMargin: 0 !default;
-$vui-emphasis-rightMargin: 0 !default;
-$vui-emphasis-bottomMargin: 20px !default;
-$vui-emphasis-leftMargin: 0 !default;
-
-$vui-help-fontColor: #666666 !default;
-$vui-help-fontSize: 11px !default;
-$vui-help-fontStyle: italic !default;
-$vui-help-fontWeight: normal !default;
-$vui-help-lineHeight: 20px !default;
-$vui-help-topMargin: 0 !default;
-$vui-help-rightMargin: 0 !default;
-$vui-help-bottomMargin: 20px !default;
-$vui-help-leftMargin: 0 !default;
-
-@mixin vui-typography(
-		$fontColor: $vui-fontColor,
-		$fontFamily: $vui-fontFamily,
-		$fontSize: $vui-fontSize,
-		$lineHeight: $vui-lineHeight
-	) {
-
-	color: $fontColor;
-	font-family: $fontFamily;
-	font-size: $fontSize;
-	font-weight: normal;
-	line-height: $lineHeight;
 
-}
-
-@mixin vui-typography-emphasis(
-		$fontColor: $vui-emphasis-fontColor,
-		$fontStyle: $vui-emphasis-fontStyle,
-		$topMargin: $vui-emphasis-topMargin,
-		$rightMargin: $vui-emphasis-rightMargin,
-		$bottomMargin: $vui-emphasis-bottomMargin,
-		$leftMargin: $vui-emphasis-leftMargin,
-		$baseFontSize: $vui-fontSize
-	) {
-
-	color: $fontColor;
-	font-style: $fontStyle;
-	margin: pxToEm($topMargin, $baseFontSize)
-			pxToEm($rightMargin, $baseFontSize)
-			pxToEm($bottomMargin, $baseFontSize)
-			pxToEm($leftMargin, $baseFontSize);
+@import 'bower_components/vui-colors/colors';
+@import 'px-to-rem';
+@import 'fonts';
+
+$vui-base-fontSize: 19px;
+
+$vui-fontColor: $vui-color-ferrite !default;
+$vui-fontFamily: 'Lato', 'Lucida Sans Unicode', 'Lucida Grande', sans-serif !default;
+$vui-fontSize: px-to-rem(19px, $vui-base-fontSize) !default;
```

Comment:
Cool yeah with the history it makes sense. I'd lean towards ditching it then to make the code clearer and then adding it back in the future if we end up needing it.